x-pack/test/performance/config.synthetics.ts

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Url from 'url';
+import { FtrConfigProviderContext } from '@kbn/test';
+import { run as playwrightRun } from '@elastic/synthetics';
+
+import './tests/synthetics';
+
+// These "secret" values are intentionally written in the source. We would make the APM server accept annonymous traffic if we could
+const APM_SERVER_URL = 'https://2fad4006bf784bb8a54e52f4a5862609.apm.us-west1.gcp.cloud.es.io:443';
+const APM_PUBLIC_TOKEN = 'Q5q5rWQEw6tKeirBpw';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const kibanaCommonTestsConfig = await readConfigFile(
+    require.resolve('../../../test/common/config')
+  );
+  const xpackFunctionalTestsConfig = await readConfigFile(
+    require.resolve('../../../x-pack/test/functional/config')
+  );
+
+  return {
+    ...kibanaCommonTestsConfig.getAll(),
+    esTestCluster: xpackFunctionalTestsConfig.get('esTestCluster'),
+    kbnTestServer: {
+      ...xpackFunctionalTestsConfig.get('kbnTestServer'),
+      env: {
+        ELASTIC_APM_ACTIVE: 'true',
+        ELASTIC_APM_CONTEXT_PROPAGATION_ONLY: 'false',
+        ELASTIC_APM_ENVIRONMENT: process.env.CI ? 'ci' : 'development',
+        ELASTIC_APM_TRANSACTION_SAMPLE_RATE: '1.0',
+        ELASTIC_APM_SERVER_URL: APM_SERVER_URL,
+        ELASTIC_APM_SECRET_TOKEN: APM_PUBLIC_TOKEN,
+        ELASTIC_APM_GLOBAL_LABELS: Object.entries({
+          ftrConfig: `x-pack/test/performance`,
+          jenkinsJobName: process.env.JOB_NAME,
+          jenkinsBuildNumber: process.env.BUILD_NUMBER,
+          prId: process.env.PR_NUMBER,
+          branch: process.env.GIT_BRANCH,
+          commit: process.env.GIT_COMMIT,
+          mergeBase: process.env.PR_MERGE_BASE,
```

Comment:
Quite a few of these are already included by the default configs when `CI=true`. Additionally, https://github.com/elastic/kibana/pull/117749 was merged so we should have correct branch information.

I believe the following can be removed:
- jenkinsJobName
- jenkinsBuildNumber
- prId
- branch
- commit
- mergeBase
- targetBranch

Reply:
For tests you might want set sampling to 100%, no?

Reply:
Sorry, should have been more clear. I am speaking only to the `ELASTIC_APM_GLOBAL_LABELS` variable.